pagerduty/resource_pagerduty_event_orchestration_path_global.go

```diff
@@ -374,6 +378,7 @@ func expandGlobalPathActions(v interface{}) *pagerduty.EventOrchestrationPathRul
 		actions.Suppress = a["suppress"].(bool)
 		actions.Suspend = intTypeToIntPtr(a["suspend"].(int))
 		actions.Priority = a["priority"].(string)
+		actions.EscalationPolicy = a["escalation_policy"].(string)
```

Comment:
I just realized that we'll to go back and change the `EscalationPolicy` prop type in the API Client repo to a string pointer ([example](https://github.com/alenapan/go-pagerduty/blob/master/pagerduty/automation_actions_runner.go#L15)) instead of a string in order to be able to pass EP actions as `null` if we want to unset it. Otherwise we'll be sending empty strings to the API and getting 400's back.
Also, have you tried running the acceptance tests locally?

Reply:
And we still need to add EP mapping to the `flattenGlobalPathActions` function in order to set resource data from the API payload.

Reply:
> I just realized that we'll need to go back and change the `EscalationPolicy` prop type in the API Client repo to a string pointer ([example](https://github.com/alenapan/go-pagerduty/blob/master/pagerduty/automation_actions_runner.go#L15)) instead of a string in order to be able to pass EP actions as `null` if we want to unset it. Otherwise we'll be sending empty strings to the API and getting 400's back. Also, have you tried running the acceptance tests locally?

I have not been able to run these locally. I can trigger specific tests within vscode and they return ok, but I can't run the whole suite from cli.

Reply:
re: flattenglobalactions

Early on last week when I was working on this I had a different folder checked out and I had implemented all of this, but in the process of rectifying my go installation undid aaaallll that work. Or undid when I merged from working directory to the fixed working directory. Anyway, it looks like I clobbered that change.